snuba/state/cache/redis/backend.py

```diff
@@ -26,6 +26,41 @@
 RESULT_WAIT = 2
 
 
+# ---- prototype stuff ----
+JsonStrBytes = bytes
+from typing import Any, cast
+
+import rapidjson as json
+
+ERROR_HEADER = b"__ERR__"
+
+
+def serialize_exception(exception: Exception) -> JsonStrBytes:
+    def _is_jsonable(item: Any) -> bool:
+        try:
+            json.dumps(item)
```

Comment:
This would be better if it returned the JSON value, so we don't JSON encode everything twice.